opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -109,7 +110,60 @@ namespace Opm
             adaptiveTimeStepping.reset( new AdaptiveTimeStepping( param_, terminal_output_ ) );
         }
 
-        output_writer_.writeInit();
+
```

Comment:
> the 4.th grid implementation in opm

I'll not go into this further other than to say that you are underestimating the number of grid implementations...